src/isar/state_machine/state_machine.py

```diff
@@ -80,9 +79,9 @@ def __init__(
         self.sleep_time = sleep_time
 
         self.mission_in_progress: bool = False
-        self.current_mission: Mission = Mission(steps=[])
+        self.current_mission: Mission = Mission(tasks=[])
```

Comment:
Should not `self.current_mission` be initialized to `None` before `start_mission()` is called (which populates the `current_mission`, and also be set to `None` again when the mission is done in `reset_state_machine()`?